native-addons-are-close-to-being-on-par-with-js-modules.md

```diff
@@ -0,0 +1,113 @@
+# Native Addons Are Close To Being On Par With JS Modules
+
+Native addons have been a part of Node.js since the very beginning. So have JS modules 😉 So, what does it mean that they are now close to being "on-par"? Why only now?
+
+Let's first look at the similarities between native adons and JS modules. Both can be `require()`-ed:
+
+```JS
+// A JS module can be loaded from a package:
+const jsModulePackage = require('my-js-module');
+// It can also be loaded from a specific file:
+const jsModuleFile = require('./my-js-module.js');
+
+// A native addon can also be loaded from a file:
+const nativeAddonFile = require('./build/Release/my-native-addon.node');
+```
+
+Both return a JavaScript object containing useful functionality which would otherwise be completely unavailable, or which would otherwise take a long time to implement.
+
```

Comment:
Maybe these sentences can be unified. Each paragraph, looks slightly better if it has at least two sentences.